src/pages/Script/ScriptsComp.tsx

```diff
@@ -30,6 +32,7 @@ export const ScriptTransactions = ({ page, size }: { page: number; size: number
   const { codeHash, hashType } = useParams<{ codeHash: string; hashType: string }>()
 
   const [transactionsEmpty, setTransactionsEmpty] = useState(false)
+  const [restrict, setRestrict] = useState(false)
```

Comment:
The `restrict` could be loaded from the URL by `useSearchParams` so users can share this page within/without `restricted mode`

Reply:
We can use

```ts
const { mode } = useSearchParams('mode')
const isRestricted = mode === 'restricted
```
instead of updating the state manually

e.g. https://github.com/nervosnetwork/ckb-explorer-frontend/blob/d01174a9024ce3faf987e8afd63f9346607ff710/src/pages/Xudt/index.tsx#L23